hangmanlunch.rb

```diff
@@ -0,0 +1,208 @@
+require "colorize"
+
+class Hangman
+
+  def initialize
+    @answer_array = populate_answer
+    @guess_array = populate_guess_array
+    @paper = []
+    @top_row =      "|    _________"
+    @second_row =   "|     |/      "
+    @middle_row =   "|     |       "
+    @base_row =     "| ____|___ word: #{@guess_array.join(" ")}"
+    @misses = 0
+    @incorrect_array =[]
+
+    #remove this line to actually play-- leaving it for testing purposes
+    puts "Answer: " + @answer_array.join
+
+    start
+  end
+
+  # create paper
+  def draw_paper
+    @paper.each do |row|
+      puts row
+    end
+  end
+
+  def start
+    puts "\nLet's play Hangman!!\n"
+
+    #this kicks off the game loop in "play" method below
+    @now_playing = true
+
+    # creates empty scaffold (in the paper array)
+    @paper.push(@top_row, @second_row, @base_row)
+
+    5.times do
+      @paper.insert(2, @middle_row)
+    end
+
+    # draws initial empty scaffold
+    draw_paper
+
+    #starts asking for guesses
+    play
+  end
+
+  def play
+
+    while @now_playing
+      # asks for user guess and checks their guess
+      get_guess
+      check_guess(@letter_guess)
+    end
+  end
+
+
+  def get_guess
+    puts "\nGuess a letter! Or you can give up and type 'quit'."
+    # asks for user input (i.e. their letter guess)
+    @letter_guess = gets.chomp.downcase
+
+    acceptable_range = ("a".."z")
+
+    # allows user to quit the game
+    if @letter_guess == "quit"
+      exit
+
+    # controls for unexpected user input
+    elsif acceptable_range.include?(@letter_guess) == false
+      puts "We're working with letters here. Come on...".red.bold
+    end
+    return @letter_guess
+  end
+
+  def check_guess(guess)
```

Comment:
Before I get into this method's functionality, I wanna compliment yinz on your commenting. There's obviously a lot going on here, but I'm able to follow the method's logic because of the comments. Well done.